### ISSUE TITLE
Display a horizontal line when using Vertical Space tool

### ISSUE THREAD

**Issue by redtrumpet** [xournalpp/xournalpp#2843]
Hey, I think it would be useful if the Vertical Space tool would display a thin horizontal line, so that one sees if all written letters will be selected.

Sometimes when a line is long I miss some letters, and have to start over.

**Comment by Technius** [comment]
This can be done as part of #3334.

**Comment by rodrigo103** [comment]
The PR is merged. Should this issue be closed?

**Comment by rolandlo** [comment]
I think this issue was not addressed in the PR. So the issue should stay open.

**Comment by WillNilges** [comment]
@rolandlo What do you mean by that? Do you think that a horizontal line should be displayed even before a mouse click?

**Comment by redtrumpet** [comment]
@WillNilges Yes, that was the intention of my original issue. To have a kind of "aiming guide".

**Comment by WillNilges** [comment]
Gotcha. I'll poke at this and see what I can do.

**Comment by WillNilges** [comment]
Is there another tool that draws a kind of guide in this way? My current approach, I think, is to simply make a 1px selection box appear at the cursor's position when the tool is active, but that doesn't feel quite right. I figured out how to change the mouse cursor as well.

**Comment by WillNilges** [comment]
Or I might be able to specify a fancy css cursor as stated in [`XournalppCursor.cpp`](https://github.com/xournalpp/xournalpp/blob/0ed5ed1f2da7a17a7a6371c95e57acbffeffba05/src/core/gui/XournalppCursor.cpp)?

**Comment by WillNilges** [comment]
I don't think a cursor is going to work. All I can get it to do so far is draw a really wide line over the window, which isn't bound to the document. With a little more work I got it to draw it starting at the mouse position going to the right side of the screen, but for some reason it won't let me draw anything to the left of the cursor.

**Comment by WillNilges** [comment]
@rolandlo @Technius I think I got a rough draft of a solution. Basically, I made a new cursor. Take a look, if you have a chance?
https://github.com/xournalpp/xournalpp/pull/4525
